config.cc

```diff
@@ -32,6 +32,18 @@ namespace Config {
 
 namespace
 {
+#ifdef XDG_BASE_DIRECTORY_COMPLIANCE
+  const char xdgSubdirName[] = "goldendict";
+  QDir getDataDir()
```

Comment:
Please separate the global constant (that is used in more than one function) and the function with a blank line.